frame/collective/src/lib.rs

```diff
@@ -903,41 +953,37 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 	fn do_approve_proposal(
```

Comment:
The whole comment above this extrinsic seems to be some old weight computation comment. 
Is it necessary to update it or can I delete it?

Reply:
The weight comments can all be removed 😄